app/views/shared/_header.html.erb

```diff
@@ -1,5 +1,17 @@
 <div id="header">
```

Comment:
Began addressing need for different ordering of header components based on screen size.